docs/developer-tools/runtime/services/marlowe-tx.md

```diff
@@ -5,22 +5,98 @@ title: Marlowe transaction service
 The `marlowe-tx` executable provides services related to building and submitting transactions for Marlowe contracts.
 
 ```console
-marlowe-tx : the transaction creation server of the Marlowe Runtime
-
-Usage: marlowe-tx [--chain-sync-port PORT_NUMBER]
-                  [--chain-sync-query-port PORT_NUMBER]
-                  [--chain-sync-command-port PORT_NUMBER]
-                  [--chain-sync-host HOST_NAME]
-                  [--contract-query-port PORT_NUMBER]
-                  [--contract-host HOST_NAME] [--command-port PORT_NUMBER]
-                  [-h|--host HOST_NAME] [--submit-confirmation-blocks INTEGER]
+marlowe-tx: Transaction creation server for the Marlowe Runtime.
+
+Usage: marlowe-tx [--version] [--chain-sync-port PORT_NUMBER] 
+                  [--chain-sync-query-port PORT_NUMBER] 
+                  [--chain-sync-command-port PORT_NUMBER] 
+                  [--chain-sync-host HOST_NAME] 
+                  [--contract-query-port PORT_NUMBER] 
+                  [--contract-host HOST_NAME] [--command-port PORT_NUMBER] 
+                  [-h|--host HOST_NAME] [--submit-confirmation-blocks INTEGER] 
                   [--analysis-timeout SECONDS] [--http-port PORT_NUMBER]
                   --minting-policy-cmd CMD
 
-  Marlowe runtime transaction creation server
+  The transaction creation engine for the Marlowe Runtime. This component exposes
+  a job protocol with four commands: create, apply inputs, withdraw, and submit.
+
+  Create Command
+
+  The create command will create an unsigned transaction body that starts a new Marlowe
+  contract. The options that can be configured for this command are:
+
+    • Adding a staking credential to the Marlowe script address for the contract.
+    • What addresses can be used for coin selection.
+    • Which address to send change to.
+    • Rules for role tokens in the contract. Includes rules for minting and distributing
+      role tokens in the creation transaction and rules for using existing role tokens.
+    • Adding metadata to the transaction
+    • The amount that should be deposited in the contract to cover min UTxO rules
+      (can be omitted and the runtime will compute a worst-case value automatically.
+    • The initial contract or the hash of a contract in the contract store to use as the
+      initial contract.
+
+  Apply Inputs Command
+
+  The apply inputs command will create an unsigned transaction body that advances a
+  Marlowe contract by applying transaction inputs to it. The options that can be
+  configured for this command are:
+
+    • What addresses can be used for coin selection.
+    • Which address to send change to.
+    • Adding metadata to the transaction
```

Comment:
Please add a period at the end of this bullet item.

Reply:
```suggestion
    • Adding metadata to the transaction.
```